outputs.tf

```diff
@@ -1,9 +1,44 @@
-output "null_resource_id" {
-  value       = "${null_resource.foo.id}"
-  description = "An arbitrary value that changes each time the resource is replaced."
+output "security_groups" {
+  value = concat(
+    aws_security_group.security-groups-ingress-cidr,
```

Comment:
Does `aws_security_group.security-groups-ingress-cidr` mean the same thing as `aws_security_group.security-groups-ingress-cidr[*]`?

Reply:
The splat expression iterates over the list. Without the splat it's just a list.

So this is just returning everything about the `aws_security_group` without needing to iterate and dig into a specific field like IDs.

Functionally, I think in this case those two statements would be identical.

https://www.terraform.io/docs/language/expressions/splat.html